pkg/connector/sequence.go

```diff
@@ -66,6 +71,12 @@ func (r *sequenceSyncer) List(ctx context.Context, parentResourceID *v2.Resource
 }
 
 func (r *sequenceSyncer) Entitlements(ctx context.Context, resource *v2.Resource, pToken *pagination.Token) ([]*v2.Entitlement, string, annotations.Annotations, error) {
+	_, _, err := parseWithDatabaseID(resource.Id.Resource)
+	if err != nil {
+		return nil, "", nil, err
+	}
+
+	// TODO: add entitlements for sequences
```

Comment:
this is done right?

Reply:
Right, but I need to add database name as prefix for this entitlement